arangod/GeneralServer/VstCommTask.cpp

```diff
@@ -329,6 +329,17 @@ void VstCommTask<T>::processMessage(velocypack::Buffer<uint8_t> buffer,
           << VstRequest::translateMethod(req->requestType()) << "\",\""
           << url(req.get()) << "\"";
 
+      std::string_view body = req->rawPayload();
+      if (Logger::isEnabled(LogLevel::TRACE, Logger::REQUESTS) &&
+          Logger::logRequestParameters()) {
```

Comment:
If `body` is not used outside of this `if`, let's move it inside.
```suggestion
      if (Logger::isEnabled(LogLevel::TRACE, Logger::REQUESTS) &&
          Logger::logRequestParameters()) {
        std::string_view body = req->rawPayload();        
```